content/geoip/legacy/javascript.mdx

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
Instead of adding this content, we should redirect the url to the new client-side js documentation.

Reply:
I have removed and redirected the previous WordPress urls to the new JS page.